LINUXCOMPILE.md

```diff
@@ -13,32 +13,22 @@
     make install
 
     # Fedora
-    sudo dnf install gcc gcc-c++ hwloc-devel libmicrohttpd-devel openssl-devel cmake
+    sudo dnf install gcc gcc-c++ hwloc-devel libmicrohttpd-devel libstdc++-static make openssl-devel cmake
     cmake .
     make install
 
     # CentOS
-    sudo yum install centos-release-scl cmake3 hwloc-devel libmicrohttpd-devel openssl-devel
-    sudo yum install devtoolset-4-gcc*
+    sudo yum install centos-release-scl epel-release
+    sudo yum install cmake3 devtoolset-4-gcc* hwloc-devel libmicrohttpd-devel openssl-devel make
     sudo scl enable devtoolset-4 bash
     cmake3 .
     make install
-
-    # Ubuntu 14.04
```

Comment:
I removed the whole "Ubuntu 14.04" section, because most people will build against latest Ubuntu or LTS. How to build with this old Ubuntu version is "described" in ```build_xmr-stak_docker.sh```.